Framework/Algorithms/src/FitPeaks.cpp

```diff
@@ -328,7 +375,13 @@ void FitPeaks::init() {
 
   declareProperty(PropertyNames::PEAK_MIN_HEIGHT, 0.,
                   "Minimum peak height such that all the fitted peaks with "
-                  "height under this value will be excluded.");
+                  "height under this value will be excluded. The same property is used "
+                  "for pre-checking peaks before fitting, such that all the peaks with the total Y-count "
+                  "under this value will be excluded.");
+
+  declareProperty(PropertyNames::PEAK_MIN_SIGNAL_TO_NOISE_RATIO, 0.,
```

Comment:
This is technically a breaking change as it will change the order of positional arguments passed in the python function  call (although I think the risk of someone calling `FitPeaks` with positional arguments as opposed to keyword arguments is low given the number of parameters). In general extra arguments should be added at the end, is there a specific reason for it to be here?

Reply:
Great catch! I was just trying to keep "similar things" together, but yes, I do need to move the new property to the end.

Reply:
As far as `FindBackgroundSigma`, it is not being used in the FitPeaks class. It's like a legacy parameter that you can set, but it doesn't affect anything.

Reply:
> As far as `FindBackgroundSigma`, it is not being used in the FitPeaks class. It's like a legacy parameter that you can set, but it doesn't affect anything.

Hmm, I think it would be good to remove this at some point as it could confuse the user and it doesn't do anything, but that would also be a breaking change. For now would it be possible to print a deprecation warning if this parameter is set, and suggest the user instead sets `MinimumSignalToNoiseRatio` instead?